setup/nextcloud.sh

```diff
@@ -11,88 +11,109 @@ echo "Installing Nextcloud (contacts/calendar)..."
 
 apt-get purge -qq -y owncloud* # we used to use the package manager
 
-apt_install php php-fpm \
-	php-cli php-sqlite3 php-gd php-imap php-curl php-pear curl \
-	php-dev php-gd php-xml php-mbstring php-zip php-apcu php-json \
-	php-intl php-imagick
+apt_install php php7.2-fpm \
+        php-cli php-sqlite3 php-gd php-imap php-curl php-pear curl \
+        php-dev php-gd php-xml php-mbstring php-zip php-apcu php-json \
+        php-intl php-imagick
 
 InstallNextcloud() {
 
-	version=$1
-	hash=$2
+        version=$1
 
-	echo
-	echo "Upgrading to Nextcloud version $version"
-	echo
+        echo
+        echo "Upgrading to Nextcloud version $version"
+        echo
 
         # Download and verify
-        wget_verify https://download.nextcloud.com/server/releases/nextcloud-$version.zip $hash /tmp/nextcloud.zip
-
-	# Remove the current owncloud/Nextcloud
-	rm -rf /usr/local/lib/owncloud
-
-	# Extract ownCloud/Nextcloud
-	unzip -q /tmp/nextcloud.zip -d /usr/local/lib
-	mv /usr/local/lib/nextcloud /usr/local/lib/owncloud
-	rm -f /tmp/nextcloud.zip
-
-	# The two apps we actually want are not in Nextcloud core. Download the releases from
-	# their github repositories.
```

Comment:
Why don't we install these apps for version 18?

Reply:
it's later in the code